dvc/command/base.py

```diff
@@ -52,5 +56,7 @@ def run(self):
 
 
 class CmdBaseNoRepo(CmdBase):
+    check_updates = False
+
```

Comment:
I don't understand, why don't we want to check for updates on no-repo commands?

Reply:
As it is implemented right now, `Updater` uses `.dvc/tmp`. And, I haven't changed any implementation, Updater runs only on `CmdBase`, never on `CmdBaseNoRepo`.